modules/ossm-upgrading-operator.adoc

```diff
@@ -4,25 +4,35 @@
-
-Updating the Operator does not affect the version of any component other than the Operator. Updating the Operators does _not_ update the `ServiceMeshControlPlane` version or deployments.
+The version of the Operator does *not* determine the version of your service mesh. The version of your deployed control plane determines your version of Service Mesh.
 ====
 
-When you installed your Operators, you selected an *Update Channel* and an *Approval Strategy*. Those two settings determine when and how your Operators are updated.
+Because the {SMProductName} Operator supports multiple versions of the control plane, updating the {SMProductName} Operator does _not_ update the `spec.version` value of your deployed `ServiceMeshControlPlane`. Also note that the `spec.version` value is a two digit number, for example 2.2, and that patch updates, for example 2.2.1, are not reflected in the SMCP version value.
+
+Operator Lifecycle Manager (OLM) controls the installation, upgrade, and role-based access control (RBAC) of Operators in a cluster. The OLM runs by default in {product-title}. OLM queries for available Operators as well as upgrades for installed Operators.
+
+Whether or not you have to take action to upgrade your Operators depends on the settings you selected when installing them. When you installed each of your Operators, you selected an *Update Channel* and an *Approval Strategy*. The combination of these two settings determine when and how your Operators are updated.
 
 .Interaction of Update Channel and Approval Strategy
 [options="header"]
 [cols="a, a, a"]
 |====
-| |Versioned channel|"Stable" or "Preview" channel
+| |Versioned channel|"Stable" or "Preview" Channel
 |*Automatic*
-|Automatically updates Operator for minor and patch releases for that version only. Will not automatically update to the next major version (that is, from version 2.0 to 3.0). Manual change to Operator subscription required to update to the next major version.
+|Automatically updates the Operator for minor and patch releases for that version only. Will not automatically update to the next major version (that is, from version 2.0 to 3.0). Manual change to Operator subscription required to update to the next major version.
 |Automatically updates Operator for all major, minor, and patch releases.
 
 |*Manual*
 |Manual updates required for minor and patch releases for the specified version. Manual change to Operator subscription required to update to the next major version.
 |Manual updates required for all major, minor, and patch releases.
 |====
+
+When you update your {SMProductName} Operator the Operator Lifecycle Manager (OLM) will remove the old Operator pod and start a new pod. Once the new Operator pod starts, the reconciliation process will check the `ServiceMeshControlPlane` (SMCP), and if there are updated container images available for any of the control plane components, it will replace those control plane pods with ones that use the new container images.
```

Comment:
Suggested edits: 
s/will remove the old Operator pod and start a new pod/removes the old Operator pod and starts a new pod

s/the reconciliation process will check the `ServiceMeshControlPlane` (SMCP)/the reconciliation process checks the `ServiceMeshControlPlane` (SMCP)

s/it will replace those control plane pods/it replaces those control plane pods